src/components/index.ts

```diff
@@ -3,9 +3,16 @@ export { Container } from './Container'
 export { Header } from './Header'
 export { HeaderNav } from './HeaderNav'
 export { HeaderNavItem } from './HeaderNavItem'
+export {
+  FooterNavItem,
+  FooterMenuNavItem,
+  FooterSupportNavItem,
```

Comment:
Make sure `FooterMenuNavItem` and `FooterSupportNavItem` components exist or are exported from the file.